README.md

```diff
@@ -33,4 +50,9 @@ s3:
 elastic:
   user: "elastic"
   password: "elastic"
+db:
+  host: "localhost"
```

Comment:
Use `127.0.0.1` to be consistent. I encountered issues since the machine tries to resolve this as ipv6 and the server is not listening to the contacted address.